### PR TITLE
[ISSUE-#4134] check namespace is exist when request naming api or config api

### PR DESCRIPTION
**Pull request by horizonzy**
## What is the purpose of the change
issue-#4134

### CODE REVIEW

File: console/src/main/java/com/alibaba/nacos/console/controller/NamespaceController.java [hunk 1]
```diff
@@ -137,6 +141,7 @@ public Boolean createNamespace(HttpServletRequest request, HttpServletResponse r
         }
         persistService.insertTenantInfoAtomic("1", namespaceId, namespaceName, namespaceDesc, "nacos",
                 System.currentTimeMillis());
+        namespaceService.addTenantId(namespaceId);
```
Comment (KomachiSion) [inline console/src/main/java/com/alibaba/nacos/console/controller/NamespaceController.java:144]: replace method name by addNamespaceId?

**Review by KomachiSion — CHANGES_REQUESTED** [review]
We may replace all tenantId with namespaceId in future. So I think use namespaceId as method name is better.

**Comment by CaiJiHuzhuo** [comment]
move console/NamespaceController.java to naming module, then config module adds two api(add,delete). when namespace change，use Http send namespaceId update data to config module.
The namespaceId is stored in both the name and config modules.
check namespaceId exists in memory.

**Comment by horizonzy** [comment]
> move console/NamespaceController.java to naming module, then config module adds two api(add,delete). when namespace change，use Http send namespaceId update data to config module.
> The namespaceId is stored in both the name and config modules.
> check namespaceId exists in memory.

Why move NamespaceController to naming module and config module? Run one module alonely?

**Comment by horizonzy** [comment]
This feature will be moved to 2.x, so close it.

**Pull request closed without merging**